api/lib/source/polkadotV0-source.js

```diff
@@ -37,6 +37,13 @@ class polkadotAPI {
     return api
   }
 
+  async getNetworkAccountInfo(address, api) {
```

Comment:
nice

Reply:
the identities will get stale. but we shouldn't care about it right now as we release the API frequently

Reply:
how about returning the result after `networkAccountReducer`? that would reduce code

Reply:
```suggestion
  async getNetworkAccountInfo(address, api, reducers) {
```

Reply:
ah, I see what you mean. And calling `networkAccountReducer` only once inside `getNetworkAccountInfo`.

Good idea

Reply:
yep

Reply:
as a guide: things we store in "store" should already be in the Lunie data model so after reducers.

Reply:
Done